Reuse existing MicroCeph and MicroOVN clusters (#259)
canonical/microcloud: doc/how-to/initialise.rst

See an example of the full initialisation process in the :ref:`Get started with MicroCloud <initialisation-process>` tutorial.

Excluding MicroCeph or MicroOVN from MicroCloud
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> If the MicroOVN or MicroCeph snap is not installed on the system that runs :command:`microcloud init`, you will be prompted with the following question:

Reviewer: ```suggestion
If the MicroOVN or MicroCeph snap is not installed on the system that runs :command:`microcloud init`, you will be prompted with the following question::
```

Sorry, missed that in my previous review. I had it in my first one though. :stuck_out_tongue_closed_eyes: 
You need two colons to introduce a code block.

Same change below as well.

See an example of the full initialisation process in the :ref:`Get started with MicroCloud <initialisation-process>` tutorial.

Excluding MicroCeph or MicroOVN from MicroCloud
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

If the MicroOVN or MicroCeph snap is not installed on the system that runs :command:`microcloud init`, you will be prompted with the following question:

>      MicroCeph not found. Continue anyway? (yes/no) [default=yes]:
> 
>      MicroOVN not found. Continue anyway? (yes/no) [default=yes]:
> 
>   If you choose ``yes``,  only existing services will be configured on all systems.
> 
>   If you choose ``no`` the setup will be cancelled.

Reviewer: ```suggestion
    MicroCeph not found. Continue anyway? (yes/no) [default=yes]:

    MicroOVN not found. Continue anyway? (yes/no) [default=yes]:

If you choose ``yes``,  only existing services will be configured on all systems.
If you choose ``no``, the setup will be cancelled.
```

Too much indentation (must be 4 spaces afaik).
Also, missing comma in the last sentence, and I would combine the last two sentences into one paragraph (which means get rid of the empty line between them).

See an example of the full initialisation process in the :ref:`Get started with MicroCloud <initialisation-process>` tutorial.

Excluding MicroCeph or MicroOVN from MicroCloud

> -----------------------------------------------

Reviewer: ```suggestion
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

This section should be a subsection of "Interactive configuration", right?
Then we'll need to use another symbol for the heading (not sure about the length of the suggestion :wink: - should be as long as the title).

> If the MicroOVN or MicroCeph snap is not installed on the system that runs :command:`microcloud init`, you will be prompted with the following question:
> 
>   .. terminal::

Reviewer: ```suggestion
If the MicroOVN or MicroCeph snap is not installed on the system that runs :command:`microcloud init`, you will be prompted with the following question::

```

No need for `terminal::` here - we use this when we want to show input and output. For only output, code style is sufficient.

     MicroCeph not found. Continue anyway? (yes/no) [default=yes]:

     MicroOVN not found. Continue anyway? (yes/no) [default=yes]:

>   #. If you choose ``yes``,  only existing services will be configured on all systems.
> 
>   #. If you choose ``no`` the setup will be aborted.

Reviewer: ```suggestion
If you choose ``yes``, only existing services will be configured on all systems.
If you choose ``no``, the setup will be stopped.
```

No need for a list here imo, the paragraph is short enough. :)

All other systems must have at least the same set of snaps installed as the system that runs :command:`microcloud init`, otherwise they will not be available to select from the list of systems.
Any questions associated to these systems will be skipped. For example, if MicroCeph is not installed, you will not be prompted for distributed storage configuration.

Reusing an existing MicroCeph or MicroOVN with MicroCloud

> ---------------------------------------------------------

Reviewer: ```suggestion
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

> If some of the systems are already part of a MicroCeph or MicroOVN cluster, you may reuse this cluster when initialising MicroCloud when prompted with the following question:

Reviewer: ```suggestion
If some of the systems are already part of a MicroCeph or MicroOVN cluster, you can choose to reuse this cluster when initialising MicroCloud when prompted with the following question::
```

>   .. terminal::

Reviewer: ```suggestion
```

     "micro01" is already part of a MicroCeph cluster. Do you want to add this cluster to MicroCloud? (add/skip) [default=add]:

     "micro01" is already part of a MicroOVN cluster. Do you want to add this cluster to MicroCloud? (add/skip) [default=add]:

>   #. If you choose ``add``, MicroCloud will add the remaining systems selected for initialisation to the pre-existing cluster.
> 
>   #. If you choose ``skip``, MicroCloud will not configure the service to work with MicroCloud on any system.

Reviewer: ```suggestion
If you choose ``add``, MicroCloud will add the remaining systems selected for initialisation to the pre-existing cluster.
If you choose ``skip``, MicroCloud will not configure the service to work with MicroCloud on any system.
```

Reviewer: If you choose skip, will MicroOVN/Ceph be set up in the usual way (forming a new cluster)? Or will it not be set up at all?
That's unclear from the description.

Author: Right, I meant that it won't be set up at all. 

How about:
`If you choose ``skip``, MicroCloud will ignore the service on all systems`?

Reviewer: Or maybe:
```
If you choose ``skip``, the respective service will not be set up at all.
```
"Ignoring the service" could also be interpreted as "ignoring the existing service/cluster".

> If more than one MicroCeph or MicroOVN cluster exists among the systems, the MicroCloud initialisation will abort.

Reviewer: ```suggestion
If more than one MicroCeph or MicroOVN cluster exists among the systems, the MicroCloud initialisation will be cancelled.
```
